tests/generic_views/test_edit.py

```diff
@@ -90,6 +90,21 @@ def test_post_data(self):
         self.assertRedirects(res, '/list/authors/')
 
 
+@override_settings(ROOT_URLCONF='generic_views.urls')
```

Comment:
I wonder if this would be better organized in a new file something like `test_combined_views.py` with the view, urls, and test all in the same file?